frontend/src/kueueUtils.ts

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
hey @srtanish1992 while you re fixing tests can you move these files under the hardware profiles concepts folder